src/main/scala/tech/cryptonomic/conseil/util/JsonUtil.scala

```diff
@@ -56,7 +56,7 @@ object JsonUtil {
     lazy val emptyObject = JsonString("{}")
 
     /** add standard cleaning for input json */
-    def sanitize(s: String): String = s.filterNot(_.isControl)
+    def sanitize(s: String): String = s.filterNot(_.isControl).replaceAll("\\\\", "backslash")
```

Comment:
So it wasn't _backspace_ but backslash 😁  
Apart from useless comments...

You might want to use raw string literals as in
`replaceAll("""\\""", "backslash")`

if it looks better

Reply:
The particular problem was actually _backspace_, but it was in literal form so the earlier code did not strip it out. By replacing _backslash_ we 'disable' any control character.